src/coreanimation.cs

```diff
@@ -858,7 +858,15 @@ interface CATextLayer {
 
 	[BaseType (typeof (NSObject))]
 	[Model]
-	[Protocol (IsInformal = true)] // not informal as of iOS 10+, but removing the IsInformal value breaks when building with older SDKs (see bug #43585).
+#if IOS || TVOS
+	[Protocol (InformalSwitch = new int [] { 10, 0 })]
```

Comment:
Are there cases where protocols became informal, ie after being formal ?
Reading this the original state is unclear